README.md

```diff
@@ -1,2 +1,26 @@
 # chart-postgresql
+[![Build Status](https://dev.azure.com/hmcts/PlatformOperations/_apis/build/status/chart-postgresql)](https://dev.azure.com/hmcts/PlatformOperations/_build?definitionId=856)
 This chart is intended for adding the postgres sql databases.
+
+## Prerequisites
+
+- To use this chart, you need a postgres Flexible server created beforehand.
+- Please see [this](https://github.com/hmcts/cnp-flux-config/blob/master/apps/sscs/preview/aso/sscs-postgres.yaml) link where the Flexible server is deployed using ASO.
```

Comment:
1. use permalinks in docs
2. avoid this and here, and include the destination in text
3. a pull request example would be better as it would show all files changes

```suggestion
- Please see [sscs-postgres.yaml](https://github.com/hmcts/cnp-flux-config/blob/0a747cb190aefdd5cd881e5f10b62c99a607b061/apps/sscs/preview/aso/sscs-postgres.yaml) as an example.
```